distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -281,10 +281,24 @@ public void testReadDerivedStorage() throws IOException {
   @Test
   public void testMetacardTransformersFromRest() {
     String id = ingestGeoJson(getFileContent(JSON_RECORD_RESOURCE_PATH + "/SimpleGeoJsonRecord"));
+    String title = "myTitle";
+    String geoCoordinates = "30.0 10.0";
+
+    String idXPath = "/metacard[@id='" + id + "']";
+    String titleXPath = "/metacard/string[@name='title'][value='" + title + "']";
+    String geoXPath = "/metacard/geometry";
+    String geoCoordsXPath = geoXPath + "//Point[pos='" + geoCoordinates + "']";
 
     String url = REST_PATH.getUrl() + id;
     LOGGER.info("Getting response to {}", url);
-    when().get(url).then().log().all().assertThat().body(hasXPath("/metacard[@id='" + id + "']"));
+
+    // tests that metacard properties are properly set
+    when().get(url).then().log().all().assertThat().body(hasXPath(idXPath));
```

Comment:
❗️ In another quick pass, there are a couple more things that are concerning:
1. You're making a GET request for every one of these assertions, which is a lot of unnecessary overhead. You should move this logic around to where you're making one GET request and doing all your `assertThat`s on the one response.
2. I'm guessing the `.log().all()` logs the whole response. We either need to do that at a DEBUG log level or not do it at all. Otherwise, it's going to spam the itest logs.

Reply:
Good point, I overlooked that. I think I can move all the XPath checks in a single `allOf` matcher.